examples/blocks/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Folders
+.idea
+/node_modules
```

Comment:
Let's remove the comment and `.idea`. That should be in your global gitignore. Please build a gitignore to easily manage your dependencies [as other examples](https://github.com/tastejs/todomvc/blob/master/examples/angularjs/.gitignore).